apps/photos/src/utils/common/apiUtil.ts

```diff
@@ -45,7 +45,7 @@ export const getPublicCollectionThumbnailURL = (id: number) => {
 };
 
 export const getUploadEndpoint = () => {
-    const endpoint = process.env.NEXT_PUBLIC_ENTE_ENDPOINT;
+    const endpoint = process.env.NEXT_PUBLIC_ENTE_UPLOAD_ENDPOINT;
```

Comment:
What's this url change? As someone  without much understanding of the code base, I think based on flag we would choose different values

Reply:
so, I updated the add a new variable -> `NEXT_PUBLIC_ENTE_UPLOAD_ENDPOINT`, earlier it was api was never used and was pointing to the ENTE_ENDPOINT as default value.